### PR TITLE
group resources

### LINKED ISSUE #42: Group resources page.
* The goal of this task is to implement a page with displayed resources for a given group.
* This page should look similar to the one in the vo section and it should use some common components.

### PR DESCRIPTION
**Pull request by DanoFecko**
Resolves #42 
Basically the same functionality as in VO resources

### CODE REVIEW

File: src/app/vos/group-detail-page/group-resources/group-resources.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div>
+  <h1 class="page-subtitle">{{'VO_DETAIL.RESOURCES.TITLE' | translate}}</h1>
```
Comment (Vojtech-Sassmann) [inline src/app/vos/group-detail-page/group-resources/group-resources.component.html]: Could you please duplicate the translation values? It is better to have those values duplicated.

Comment (DanoFecko) [reply src/app/vos/group-detail-page/group-resources/group-resources.component.html]: done

**Comment by Vojtech-Sassmann** [comment]
@DanoFecko Could you please rebase the branch?

**Comment by DanoFecko** [comment]
@Vojtech-Sassmann rebase done

**Comment by DanoFecko** [comment]
@Vojtech-Sassmann  I rebased it again, but it doesnt go through travis, because of missing semicolon in file I didnt change (app-detail.comp.ts).

**Pull request merged**